calicoctl/commands/node/run.go

```diff
@@ -141,6 +142,13 @@ Options:
                            within the Docker containers that have been networked
                            by the Calico driver.
                            [default: cali]
+     --use-docker-networking-meta-data
```

Comment:
Actually here is what I'm liking right now
```
Extract the Calico-namespaced Docker container labels
(org.projectcalico.label.*) and apply them to the
container endpoints for use with Calico policy.
When this option is enabled traffic must be
explicitly allowed by configuring Calico policies
and Calico profiles are disabled.
```
Probably not necessary to say what profiles are used for, just that they are disabled.